src/spacefinder/liveblog-adverts.ts

```diff
@@ -31,34 +31,28 @@ const AD_SPACE_MULTIPLIER = 2;
 
 let AD_COUNTER = 0;
 let WINDOWHEIGHT: number;
+let IS_SERVER_SIDE_MODE = false;
```

Comment:
I'd usually avoid adding additional top-level mutable state, but I assume this will go away once we remove client-side initial insertion? In which case I think this is probably fine.

Reply:
Yeah agreed. This is indeed a temporary measure and will go away once we remove client-side initial insertion